content/pt-br/docs/tasks/configure-pod-container/migrate-from-psp.md

```diff
@@ -0,0 +1,389 @@
+   ```
+2. Compare esses Pods em execução com a especificação original do Pod, 
+   para determinar se a `PodSecurityPolicy` modificou o Pod. Para pods criados pelo 
+   [recurso de cargas de trabalho](/docs/concepts/workloads/controllers/)
+   você pode comparar o pod com o `PodTemplate` no controlador de recurso. 
+   Se alguma alteração for identificada, o Pod original ou `PodTemplate` 
+   deve ser atualizado com a configuração desejada.
+   Os campos para revisar são:
+   - `.metadata.annotations['container.apparmor.security.beta.kubernetes.io/*']` 
+     (Substitua * pelo nome de cada contêiner)
+   - `.spec.runtimeClassName`
+   - `.spec.securityContext.fsGroup`
+   - `.spec.securityContext.seccompProfile`
+   - `.spec.securityContext.seLinuxOptions`
+   - `.spec.securityContext.supplementalGroups`
+   - Em contêneres, sob `.spec.containers[*]` e `.spec.initContainers[*]`:
+       - `.securityContext.allowPrivilegeEscalation`
+       - `.securityContext.capabilities.add`
+       - `.securityContext.capabilities.drop`
+       - `.securityContext.readOnlyRootFilesystem`
+       - `.securityContext.runAsGroup`
+       - `.securityContext.runAsNonRoot`
+       - `.securityContext.runAsUser`
+       - `.securityContext.seccompProfile`
+       - `.securityContext.seLinuxOptions`
+3. Crie a nova `PodSecurityPolicies`. Se quaisquer `Roles` ou `ClusterRoles` estão 
+   permitindo `use` em todas as PSPs, isso poderia ocasionar que um novo PSPs 
+   fosse usado em vez de suas contrapartes mutantes.
+4. Altere sua autorização, para permitir o acesso aos novos PSPs. No RBAC isso 
+   significa alterar todas `Roles` ou `ClusterRoles`, que dão a permissão `use` 
+   no PSP original, para dar a permissão também ao PSP alterado.
+5. Verifique: após algum tempo, execute novamente o comando da etapa 1 
+   para ver se algum dos Pods ainda está usando os PSPs originais. 
+   Observe que os pods precisam ser recriados depois que as novas políticas 
+   foram implantadas, antes que elas possam ser totalmente verificadas.
+6. (opcional) Depois de verificar se os PSPs originais, não estão mais em uso, 
+   você pode exclui-los.
+
+## 3. Alterar Namespaces {#update-namespaces}
+
+As etapas a seguir precisarão ser executadas em cada namespace do cluster. Comandos 
+referenciados nessas etapas, usam a variável `$NAMESPACE` 
+para referir-se ao namespace sendo alterado.
+
+### 3.a. Identifique um nível apropriado de segurança de Pod {#identify-appropriate-level}
+
+Comece a revisar os [Padrões de Segurança de Pod](/docs/concepts/security/pod-security-standards/) 
+e familiarize-se com os 3 níveis diferentes.
+
+Existem várias maneiras de escolher um nível de segurança de Pod para o seu namespace:
+
+1. **Por requisitos de segurança para o namespace** - Se você estiver familiarizado 
+   com o nível de acesso esperado para o namespace, você pode escolher um nível 
+   apropriado com base nesses requisitos, semelhante a como alguém poderia 
```

Comment:
Sounds better to me "semelhante a como se poderia abordar" to avoid "cacofonia" in "como alguém"